<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="c.ExtentReportsClassTest" duration-ms="188" started-at="2018-01-31T12:20:30Z" finished-at="2018-01-31T12:20:30Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="188" started-at="2018-01-31T12:20:30Z" finished-at="2018-01-31T12:20:30Z">
      <class name="c.ExtentReportsClassTest">
        <test-method status="PASS" signature="startReport()[pri:0, instance:c.ExtentReportsClassTest@7f63425a]" name="startReport" is-config="true" duration-ms="10" started-at="2018-01-31T17:50:30Z" finished-at="2018-01-31T17:50:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method status="FAIL" signature="failTest()[pri:0, instance:c.ExtentReportsClassTest@7f63425a]" name="failTest" duration-ms="6" started-at="2018-01-31T17:50:30Z" finished-at="2018-01-31T17:50:30Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]at org.testng.Assert.fail(Assert.java:96)at org.testng.Assert.failNotEquals(Assert.java:776)at org.testng.Assert.assertTrue(Assert.java:44)at org.testng.Assert.assertTrue(Assert.java:54)at c.ExtentReportsClassTest.failTest(ExtentReportsClassTest.java:34)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- failTest -->
        <test-method status="PASS" signature="getResult()[pri:0, instance:c.ExtentReportsClassTest@7f63425a]" name="getResult" is-config="true" duration-ms="3" started-at="2018-01-31T17:50:30Z" finished-at="2018-01-31T17:50:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="passTest()[pri:0, instance:c.ExtentReportsClassTest@7f63425a]" name="passTest" duration-ms="1" started-at="2018-01-31T17:50:30Z" finished-at="2018-01-31T17:50:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- passTest -->
        <test-method status="PASS" signature="getResult()[pri:0, instance:c.ExtentReportsClassTest@7f63425a]" name="getResult" is-config="true" duration-ms="0" started-at="2018-01-31T17:50:30Z" finished-at="2018-01-31T17:50:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="as()[pri:0, instance:c.ExtentReportsClassTest@7f63425a]" name="as" is-config="true" duration-ms="380" started-at="2018-01-31T17:50:30Z" finished-at="2018-01-31T17:50:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- as -->
      </class> <!-- c.ExtentReportsClassTest -->
    </test> <!-- Command line test -->
  </suite> <!-- c.ExtentReportsClassTest -->
</testng-results>
